deployment/hasura/metadata/databases/tables/actions/action_run.yaml

```diff
@@ -0,0 +1,47 @@
+table:
+  name: action_run
+  schema: actions
+configuration:
+  custom_name: "action_run"
+object_relationships:
+  - name: action_definition
+    using:
+      foreign_key_constraint_on: action_definition_id
+select_permissions:
+  - role: aerie_admin
+    permission:
+      columns: "*"
+      filter: {}
+      allow_aggregations: true
+  - role: user
+    permission:
+      columns: "*"
+      filter: {}
+      allow_aggregations: true
+  - role: viewer
+    permission:
+      columns: "*"
+      filter: {}
+      allow_aggregations: true
+insert_permissions:
+  - role: aerie_admin
+    permission:
+      columns: "*"
+      check: {}
+      set:
+        created_by: "x-hasura-user-id"
+  - role: user
+    permission:
+      columns: "*"
```

Comment:
I'd replace `columns` with a specific list of columns, as users should _not_ be able to set the `status` column on-insert